src/store/eventing/__tests__/status-test.ts

```diff
@@ -0,0 +1,47 @@
+import eventManager from '../../../common/EventManager';
+import { AppState } from '../../types';
+import { handleSetStatusAction } from '../status';
+
+jest.mock('../../../common/EventManager');
+
+describe('store/eventing/status', () => {
+    test('should emit creator_status_change with status and type', () => {
+        handleSetStatusAction(
+            {
+                common: {
+                    mode: 'region',
+                },
+                creator: {
+                    status: 'init',
+                },
+            } as AppState,
+            {
+                common: {
+                    mode: 'region',
+                },
+                creator: {
+                    status: 'started',
+                },
+            } as AppState,
```

Comment:
I think we need to firm up our design for how annotations sdk mode will work with the discoverability feature. Will annotations sdk mode be `region` by default? Or will we enable region creation when annotations sdk mode is `none`?

Reply:
Ideally, I think the Annotations SDK mode should be "none" by default and that the mode should determine which creation flow is active, where "none" would continue to mean none. The Preview SDK can then set the mode it desires once the annotator has been initialized.

Reply:
I think this makes sense, we've tried to abstract away the funky preview sdk button states from annotations sdk. So  from the discoverability perspective, we'll need to initialize Annotations sdk with mode=region and update the FSM to set the annotations sdk mode to `region` each time we toggle off the explicit annotation mode. 

@ChenCodes this means the logic to turn off `pointer-events` in the text layer when in explicit region mode will need to live in preview sdk now